src/Illuminate/Container/Container.php

```diff
@@ -640,7 +640,7 @@ public function makeWith($abstract, array $parameters = [])
     /**
      * Resolve the given type from the container.
      *
-     * @param  string  $abstract
+     * @param  string|callable $abstract
```

Comment:
```suggestion
     * @param  string|callable  $abstract
```

Just needed an extra space here to follow code style

Reply:
oh! well spotted.